include/picongpu/CMakeLists.txt

```diff
@@ -247,6 +249,19 @@ if(ADIOS_FOUND)
     set(LIBS ${LIBS} ${ADIOS_LIBRARIES})
 endif(ADIOS_FOUND)
 
+################################################################################
```

Comment:
Please move this block down, above the `libSplash` block.

Reply:
Just to be sure, you're talking about the ADIOS1 block?